Modules/StudyProgramme/classes/class.ilStudyProgrammeChangeDeadlineGUI.php

```diff
@@ -21,7 +20,7 @@ class ilStudyProgrammeChangeDeadlineGUI
     protected ilObjUser $user;
     protected Factory $input_factory;
     protected Renderer $renderer;
-    protected ServerRequest $request;
+    protected Psr\Http\Message\ServerRequestInterface $request;
```

Comment:
Can we get rid of this completely in favour of `ILIAS\HTTP\Wrapper\RequestWrapper`?

Reply:
Not yet, the 'withRequest' method typehints on that type.